pants.toml

```diff
@@ -173,6 +173,12 @@ java_parser = "src/python/pants/backend/java/dependency_inference/java_parser.lo
 # Has the same isolation requirements as `java_parser`.
 scala_parser = "src/python/pants/backend/scala/dependency_inference/scala_parser.lockfile"
 
+[junit]
+lockfile = "src/python/pants/backend/java/test/junit.default.lockfile.txt"
+
+[google-java-format]
+lockfile = "src/python/pants/backend/java/lint/google_java_format/google_java_format.default.lockfile.txt"
```

Comment:
Ah. This is why we have the `generate_all_lockfile.sh` script for Python. It will configure this as a CLI option when running `generate-lockfiles`, such that our normal `pants.toml` isn't polluted but we can still generate the defaults when needed.